docs/application/dotnet/get-started/mobile/first-app.md

```diff
@@ -1,184 +1,126 @@
-file](../../../vstools/tools/manifest-editor.md). The Visual Studio
-template generates the manifest file (`tizen-manifest.xml`) to the top
-level of the &lt;projectname&gt;.Tizen project (if you create projects
-with mobile, TV, or wearable profiles, a separate manifest file is
-generated for each profile).
-
-For this example application, the default manifest is sufficient. If you
-want to make any changes in the application, such as changing the
-application icon or installing resources that are used by the
-application at runtime, see [Packaging Your Application](#packaging-your-application).
+![Project with Mobile, TV, and Wearable profiles](media/vs_solution_explorer_mobile.png)
+
+-   The **\<projectname\>** project contains the Xamarin.Forms code shared across platforms.
+-   If you select the common profile in the Tizen Project Wizard, a common project titled **\<projectname\>.Tizen** is added. It contains code to instantiate your common application within the Tizen framework.
+-   If you select the mobile profile in the Tizen Project Wizard, a mobile project titled **\<projectname\>.Tizen.Mobile** is added. It contains code to instantiate your mobile application within the Tizen framework.
+
+If you are already familiar with Xamarin.Forms, this project has the same structure as a Xamarin.Forms portable application. The **\<projectname\>** project is the portable class library and the others are the platform-specific projects; however, in Tizen .NET, only the Tizen platform-specific project is generated.
+
+The `.cs` file in the portable project already contains simple Xamarin.Forms code that makes a basic UI.
+
+## Build Your Application
+
+After you have created the application project, you can implement the required features. In this example, only the default features from the project template are used, and no code changes are required.
+
+When your application code is ready, build the application. The building process performs a validation check and compiles your files. You must sign the application package with an author certificate when building the application. If you have not yet registered a Tizen certificate in Visual Studio, see [Certificate Manager](../../../vstools/tools/certificate-manager.md).
+
+There are two different ways to build the application:
+
+-   In the Visual Studio menu, select **Build \> Build Solution**.
+-   In the **Solution Explorer** view, right-click the solution name and select **Build**.
+
+Tizen .NET applications are always deployed as installed packages. The package files have the `.tpk` file extension, and the process of generating a package is controlled by the [manifest file](../../../vstools/tools/manifest-editor.md). The Visual Studio template generates the manifest file (`tizen-manifest.xml`) to the top level of the \<projectname\>.Tizen project (if you create projects with mobile, TV, or wearable profiles, a separate manifest file is generated for each profile).
```

Comment:
Referencing the update in https://github.com/Samsung/tizen-docs/pull/1018 : 

```suggestion
Tizen .NET applications are always deployed as installed packages. The package files have the `.tpk` file extension, and the process of generating a package is controlled by the [manifest file](../../../vstools/tools/manifest-editor.md). The Visual Studio template generates the manifest file (`tizen-manifest.xml`) to the top level of the \<projectname\>.Tizen project (if you create projects with Mobile, TV, or Wearable profiles, a separate manifest file is generated for each profile).
```

Reply:
The original style is correct. I restored this PR and other related PRs.